target/linux/ramips/image/mt7621.mk

```diff
@@ -9,6 +9,29 @@ DEFAULT_SOC := mt7621
 KERNEL_DTB += -d21
 DEVICE_VARS += ELECOM_HWNAME LINKSYS_HWNAME
 
+define Build/beeline-sb-flash-header
+	echo -ne "$$(echo 000001001c0000000000420000000000 | sed \
+		's/../\\x&/g')" | dd of=$@.tmp bs=1 count=16 \
```

Comment:
Why do you use the sed and not directly this?
``
echo "\x00\x00\x01\x00\x1c\x00\x00\x00\x00\x00\x42\x00\x00\x00\x00\x00"
``

Reply:
@hauke in this place - to keep the string length less than 74 characters. In other places - for unification. Please let me know if it's strict requirement (hexadecimal escape sequence) and I'll think about how it can be done.

Reply:
The 74 characters length limit is used to improve readability. If it is better readable if it gets longer I am fine with it.

Reply:
Ok, I'll keep that in mind. Fortunately, long lines are no longer necessary after the latest changes. Otrx utility and  -M key turned out to be very useful for this case.